src/components/avatar/Avatar.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import * as RadixAvatar from '@radix-ui/react-avatar';
+import clsx from 'clsx';
+
+import { Button } from '../buttons';
+import { Icon } from '../icon';
+
+export interface AvatarProps extends React.ComponentPropsWithoutRef<typeof RadixAvatar.Root> {
+  src?: string;
+  alt?: string;
+  fallback?: string;
+  size?: 'sm' | 'md' | 'lg' | 'xl';
+  isEditable?: boolean;
+  onEdit?: () => void;
+}
+
+export const Avatar = React.forwardRef<HTMLSpanElement, AvatarProps>(
```

Comment:
forwardRef ist seit React 19 deprecated: https://react.dev/reference/react/forwardRef

Und falls ihr in früheren React Versionen damit gearbeitet hättet wäre es auch wichtig gewesen bei Button, Input, etc. das forwardRef zu setzen.